libs/http/include/http/json_client.hpp

```diff
@@ -89,86 +90,5 @@ class JsonClient
   ClientPtr client_;
 };
 
-/**
```

Comment:
Haha isn't this the second time you have proposed this change. I would simply keep these in since they are effectively function aliases

Reply:
That's very possible. These things do bug me.